src/views/Post/PostCreatePage/Category/HashTag/HashTag.jsx

```diff
@@ -21,7 +21,7 @@ const HashTag = ({ margin, onChange }) => {
       e.target.value = '';
     }
 
-    if (e.key === 'Enter' && (e.target.value.length > 5 || tags.length > 6)) {
+    if (e.key === 'Enter' && (e.target.value.length > 5 || tags.length >= 6)) {
```

Comment:
if 문에 들어가는 조건이 의미를 파악하기에 길어지면 변수로 쓰는 게 좋다고 배웠습니다.
아래는 개인적으로 사용하는 예시입니다.

```suggestion
    if (userInputEnter && isOverMaxLength) {
```